LightBlue.MultiHost.Core/Configuration/MultiHostRoot.cs

```diff
@@ -0,0 +1,24 @@
+﻿using LightBlue.MultiHost.Core.IISExpress;
+using LightBlue.MultiHost.Core.Runners;
+using LightBlue.Setup;
+
+namespace LightBlue.MultiHost.Core.Configuration
+{
+    public class MultiHostRoot
```

Comment:
Planning on using this composition root for both the wpf terminal and the windows service http://blog.ploeh.dk/2011/07/28/CompositionRoot/